setup.py

```diff
@@ -14,9 +15,9 @@
     'spacy==2.2.4',
     'tensorboard>=2.1.0',
     'tensorflow>=2.2.0rc1',
-    'tokenizers==0.5.2',
+    'tokenizers>=0.5.2',
     'tqdm==4.45.0',
-    'transformers==2.7.0'
+    'transformers>=2.7.0'
```

Comment:
can we force 2.8.0 already?

Reply:
My part of the code can, I don't know if it will cause issues in the rest (I don't think so..)

Reply:
Yeah.. lets try to move to 2.8.0 as it contains some important new functions for T5